pkgs/applications/emulators/c64-debugger/default.nix

```diff
@@ -0,0 +1,90 @@
+{ fetchgit, lib, pkgs, stdenv }:
+
+stdenv.mkDerivation {
+  name = "c64-debugger";
+  version = "0.64.58.6";
+
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/c64-debugger/code";
+    rev = "f97772e3f5c8b4fa99e8ed212ed1c4cb1e2389f1";
```

Comment:
The repository does not have anything like human-readable tags or releases?

Reply:
No. The project does not follow very good development practices in anything. Basically it looks like every commit made in the repository is a new release.

Reply:
I found the mirror as well, but thought the original repo would be a better/safer option to use, despite the lack of tagged versions.